sdcm/db_log_reader.py

```diff
@@ -38,6 +39,7 @@ class DbLogReader(Process):
         ' | systemd-logind:',
         ' | sudo:',
         ' | dhclient[',
+        ' | cloud-init[',
```

Comment:
will we loose capability of easy debuging cloud-init where SMI is executed?

Reply:
this means it won't be in SCT log, and we won't process events from those line.

it would still be in the logs